Fix typo (#15876)
JabRef/jabref: docs/code-howtos/dependency-management.md

```

In these cases, first determine if adding the new 3rd party module is really needed/intended.

> If yes, there are three levels of patching that can be performed:

Reviewer: ```suggestion
If yes, there are three levels of mapping/patching that can be performed:
```

The first point below is not "patching" in that sense. But it makes sense to talk about "three things" if there are three bullet points.

> 1. Add missing mappint to `modules.properties`:

Reviewer 2: ```suggestion
1. Add missing mapping to `modules.properties`:
```

>    In case the module is a real java module, but not loaded by gradle, you need it to put the mapping into `gradle/modules.properties`.

Reviewer 2: ```suggestion
   In case the module is a real Java module, but not loaded by Gradle, you need it to put the mapping into `gradle/modules.properties`.
```

Or no?

>    In case the module is a real Java module, but not loaded by Gradle, you need it to put the mapping into `gradle/modules.properties`.

Reviewer: ```suggestion
   In case the module is a real Java module, but not found by Gradle, you need to put a mapping from the _Module Name_ to the _Maven Coordinates_ (`group:name`) into `gradle/modules.properties`.
```
